src/content/configuration/externals.mdx

```diff
@@ -370,13 +457,36 @@ module.exports = {
     outputModule: true,
   },
   externalsType: 'module',
+  externals: {
+    jquery: '$',
+  },
```

Comment:
Maybe just use `externals: 'jquery'` in those cases? I can understand such usage under the `externalsType.global` section, but here could introduce confusion to users because we'll `import` from `$` now which doesn't really exist.